### PR TITLE
docs for new variant pathogenicity tools

### PR DESCRIPTION
**Pull request by sarahhunt**

### CODE REVIEW

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 1]
```diff
@@ -31,14 +31,27 @@ <h1 style="margin-top:15px">Ensembl Variation - Protein function predictions</h1
...
+<p>
 SIFT predictions are also available for cat, chicken, cow, dog, goat, horse, mouse, pig, rat, sheep and zebrafish.
 </p>
 
 <p>
-These tools are developed by external groups and we provide a brief explanation of
-the approach each takes below, and how we run it in Ensembl. For much more detail
-please see the representative papers listed <a href="#references">below</a>, and the 
-relevant publications available on each tool's website. 
+Pathogenicity scores from CADD, REVEL, MutationAssessor and MetaLR are also available for human variants.
```
Comment (ens-lgil) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Add internal links to the corresponding sections (e.g. CADD, REVEL, ...) ?

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 2]
```diff
@@ -345,13 +436,49 @@ <h2 id="references">References</h2>
     <a href="http://dx.doi.org/10.1038/nmeth0410-248" rel="external">doi:10.1038/nmeth0410-248</a></p> 
   </li>
 
+  <li>
+    <p>Dong C., Wei P., Jian X., et al. <br /> 
+    <strong>Comparison and integration of deleteriousness prediction methods for nonsynonymous SNVs in whole exome sequencing studies.</strong><br /> 
+    <em>Human Molecular Genetics</em> 24(8):2125-2137. doi:10.1093/hmg/ddu733. (2015) </br>
```
Comment (ens-lgil) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Should be `<br />`

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 3]
```diff
@@ -125,6 +138,7 @@ <h2 id="sift">SIFT</h2>
 
 <br />
 
+<h2>Results only available for human variants</h2>
```
Comment (ens-lgil) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Should we add more vertical separation above ? Or an horizontal bar ?

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html (hunk 2 above)
Comment (sarahhunt) [reply ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Thanks - fixed!

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html (hunk 1 above)
Comment (sarahhunt) [reply ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Good idea!

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html (hunk 3 above)
Comment (sarahhunt) [reply ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: I've not done this, but feel free to beautify later!

**Comment by sarahhunt** [comment]
Thanks @ens-lgil ! I've updated the link but not added the traffic light descriptions. For some, the same score can be assigned different predictions and for the two where we have assigned cut-offs ourselves, I want people to read the text rather than glance at a table.

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 4]
```diff
@@ -31,14 +31,28 @@ <h1 style="margin-top:15px">Ensembl Variation - Protein function predictions</h1
...
-please see the representative papers listed <a href="#references">below</a>, and the 
-relevant publications available on each tool's website. 
+Pathogenicity scores from <a href="#CADD">CADD</a>, <a href="#REVEL">REVEL</a>,
+<a href="#MutationAssessor"> MutationAssessor</a> and <a href="#MetaLR">MetaLR</a> are also available for human variants.
+To improve compatibility with other resources we use CADD scores from the CADD (version v1.4) project 
+while the other three are provided by dbNSFP ( version 3.5.a). 
+We display colour-coded scores for each algorithm to support immediate assessment of a set of 
+variants, but recommend the actual scores are taken into account when interpreting the impact 
+of an individual variant. This is especially important for CADD and REVEL which do not have 
+default cutoffs for different possible levels of deleteriousness. Our tables displaying the
```
Comment (at7) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: cut-offs

Comment (sarahhunt) [reply ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Thanks @at7 !

**Pull request merged**